src/AvtaleSide/steg/BeregningTilskudd/visningTilskuddsperioder/VisningTilskuddsperioderTabell.tsx

```diff
@@ -24,55 +24,68 @@ const VisningTilskuddsperioderTabell: React.FC<Properties> = ({ className }: Pro
-                            <BodyShort size="small">
-                                {avtale.tiltakstype === 'VARIG_LONNSTILSKUDD' &&
-                                    periode.status !== 'BEHANDLET_I_ARENA' && <>{periode.lonnstilskuddProsent}%</>}
-                                {(avtale.tiltakstype === 'MIDLERTIDIG_LONNSTILSKUDD' ||
-                                    avtale.tiltakstype === 'SOMMERJOBB') && <>{periode.lonnstilskuddProsent}%</>}
-                            </BodyShort>
-                            <BodyShort size="small" style={{ minWidth: '4rem' }}>
-                                {formaterPenger(periode.beløp)}
-                            </BodyShort>
-                        </div>
-                    );
-                })}
+        <>
+            <Table size={'medium'}>
+                <Table.Header>
+                    <Table.Row>
+                        <Table.HeaderCell textSize={'small'}>Tilskudd for perioder</Table.HeaderCell>
+                        {innloggetBruker.erNavAnsatt && <Table.HeaderCell textSize={'small'}>Status</Table.HeaderCell>}
+                        <Table.HeaderCell textSize={'small'}>Tilskuddsprosent</Table.HeaderCell>
+                        <Table.HeaderCell textSize={'small'}>Inntil</Table.HeaderCell>
+                    </Table.Row>
+                </Table.Header>
+                <Table.Body>
+                    {avtale.tilskuddPeriode
+                        .filter((p: TilskuddsPeriode) => p.aktiv)
+                        .map((periode: TilskuddsPeriode, index: number) => {
+                            const nyProsent: boolean =
+                                index > 0
+                                    ? avtale.tilskuddPeriode[index - 1].lonnstilskuddProsent !==
+                                      periode.lonnstilskuddProsent
+                                    : false;
+                            if (index < startIndexVisning || index > sluttIndexVisning) {
+                                return null;
+                            } else if (index !== 0 && (index === startIndexVisning || index === sluttIndexVisning)) {
+                                return (
+                                    <Table.Row key={index}>
+                                        <Table.DataCell textSize={'small'} colSpan={100}>
+                                            ...
+                                        </Table.DataCell>
+                                    </Table.Row>
+                                );
+                            }
+                            return (
+                                <Table.Row
+                                    key={index}
+                                    style={{
+                                        borderTop: nyProsent ? '2px solid gray' : 'undefined',
+                                    }}
+                                >
+                                    <Table.DataCell textSize={'small'}>
```

Comment:
Nitpick, men det er kanskje litt enklere å lese om props som er primitiver blir sendt som primitiver istedenfor som objekter?

```suggestion
                                    <Table.DataCell textSize="small">
```